server/tsuserver.py

```diff
@@ -56,6 +58,10 @@ def __init__(self):
         self.music_pages_ao1 = None
         self.backgrounds = None
         self.zalgo_tolerance = None
+        self.ipRange_bans = []
+
+        self.geoIpReader = geoip2.database.Reader('./storage/GeoLite2-ASN.mmdb')
```

Comment:
Will the server fail to run if this ASN database is not downloaded ahead of time?

Reply:
it is optional now